bundles/AdminBundle/Controller/Searchadmin/SearchController.php

```diff
@@ -478,14 +481,25 @@ public function quicksearchAction(Request $request, EventDispatcherInterface $ev
                     'fullpath' => htmlspecialchars($element->getRealFullPath()),
                     'fullpathList' => htmlspecialchars($this->shortenPath($element->getRealFullPath())),
                     'iconCls' => 'pimcore_icon_asset_default',
+                    'icon' => '',
                 ];
 
                 if ($element instanceof Asset) {
```

Comment:
I think we don't need that whole code block at all (incl. for Assets), because I think that should be now covered by `$this->addAdminStyle($element, null, $data);`. See also: https://github.com/pimcore/pimcore/blob/2f18a0f26380c56b609d937b40d2050c8d1a3e1a/models/Element/AdminStyle.php#L61

Reply:
I will need to refactor how classes are treated for `variants` where the parent object has a custom icon since one is added by a `class` and the other as a an image `src`: 
![image](https://user-images.githubusercontent.com/279826/127129772-89da9cfb-3031-4eea-b008-e7f6657bf1c4.png)